src/utilities/gql.ts

```diff
@@ -1307,6 +1475,90 @@ const gql = {
     }
   `,
 
+  GET_EXTERNAL_EVENTS: `#graphql
+    query GetExternalEvents(
+      $sourceKey: String!,
+      $derivationGroupName: String!
+    ) {
+      ${Queries.EXTERNAL_EVENT}(
+        where: {
+          source_key: {_eq: $sourceKey},
+          derivation_group_name: {_eq: $derivationGroupName}
+        }
+      ) {
+        properties
+        event_type_name
+        key
+        duration
+        start_time
+        source_key
+      }
+    }
+  `,
+
+  GET_EXTERNAL_EVENT_BY_EVENT_TYPE: `#graphql
+    query GetExternalEventByEventType($event_type_name: String!) {
+      ${Queries.EXTERNAL_EVENT}(where: {event_type_name: { _eq: $event_type_name }}) {
+        key
+        event_type_name
+        start_time
+        duration
+        properties
+      }
+    }
+  `,
+
+  GET_EXTERNAL_EVENT_TYPES: `#graphql
+    query GetExternalEventTypes {
+      external_event_types: ${Queries.EXTERNAL_EVENT_TYPES} {
+        name
+      }
+    }
+  `,
+
+  // Should be deprecated with the introduction of strict external source schemas, dictating allowable event types for given source types. But for now, this will do.
+  GET_EXTERNAL_EVENT_TYPE_BY_SOURCE: `#graphql
+    query GetExternalEventTypesBySource($derivationGroupName: String!, $sourceKey: String!) {
+      ${Queries.EXTERNAL_SOURCES} (
+        where: {
+          key: {_eq: $sourceKey},
+          derivation_group_name: {_eq: $derivationGroupName}
+        }
+      ) {
+        external_events {
+          external_event_type {
+            name
+          }
+        }
+      }
+    }
+  `,
+
+  GET_EXTERNAL_SOURCE_BY_TYPE: `#graphql
```

Comment:
Is this query still being used?

Reply:
Don't believe so - this looks like its from pre-derivation group work. Removing